src/styles/variables/duration.css

```diff
@@ -0,0 +1,9 @@
+/* Generated file. Do not edit directly. */
+:root {
+  --glide-core-duration-fast-01: 40ms;
+  --glide-core-duration-fast-02: 75ms;
+  --glide-core-duration-moderate-01: 150ms;
+  --glide-core-duration-moderate-02: 250ms;
+  --glide-core-duration-slow-01: 400ms;
+  --glide-core-duration-slow-02: 900ms;
```

Comment:
Given the name of this file, we're signaling these are general durations and not necessarily animation specific. If they're animation specific, should they go in `animation.css`?

Reply:
Design says they prefer durations and the curves being in their own collections, so we'll leave this as-is for now 🫡 .